src/utils/file-processor.js

```diff
@@ -60,6 +66,18 @@ const createMetaData = (fileName, numberOfChunks) => {
   return JSON.stringify(meta);
 };
 
+const addVersionToMeta = metaStr => `${CURRENT_VERSION}${metaStr}`;
+
+const parseMetaVersion = metaRaw => {
+  const idx = 2 * VERSION_BYTES; // 2 chars per byte in hex
+  const vStr = metaRaw.substring(0, idx);
+  const meta = metaRaw.substr(idx);
```

Comment:
Just caught this. I think it works but I'll make a change to make this consistent.